File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/bike/FindBikePayloadResponse.java
package com.lattis.ellipse.data.network.model.response.bike;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by ssd3 on 4/5/17.
 */

public class FindBikePayloadResponse {

    @SerializedName("nearest")
    List<FindBikeDataResponse> getFindNearestBikeDataResponseList;

    public List<FindBikeDataResponse> getFindNearestBikeDataResponse() {
        return getFindNearestBikeDataResponseList;
    }

    @SerializedName("available")
    List<FindBikeDataResponse> getFindAvailableBikeDataResponseList;

    public List<FindBikeDataResponse> getFindAvailableBikeDataResponse() {
        return getFindAvailableBikeDataResponseList;
    }


}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/bike/FindBikePayloadResponse.kt
package com.lattis.data.entity.response.bike

import com.google.gson.annotations.SerializedName


class FindBikePayloadResponse {

    @SerializedName("nearest")
    var findNearestBikeDataResponse: List<FindBikeDataResponse>? = null
        internal set

    @SerializedName("available")
    var findAvailableBikeDataResponse: List<FindBikeDataResponse>? = null
        internal set


}
